src/site/_includes/components/event-card/event-card.scss

```diff
@@ -45,4 +45,11 @@ $event-card-content-height: 150px;
   &__title {
     font-weight: $font-weight-bold;
   }
+
+  &__location, &__date {
+    max-width: 14em;
```

Comment:
Wat is 14em hier?

Reply:
Een quick fix omdat je geen rekening had gehouden met verschillende formaten content.